<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GovernanceList</class>
 <widget class="QWidget" name="GovernanceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_top">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="proposalSourceCombo">
         <property name="toolTip">
          <string>Select proposals to display</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filterLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Filter proposal list</string>
         </property>
         <property name="placeholderText">
          <string>Filter by Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_mn_count">
         <property name="text">
          <string>Masternode Count:</string>
         </property>
         <property name="toolTip">
          <string>Number of masternodes this wallet can vote with (masternodes for which this wallet holds the voting key)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mnCountLabel">
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="proposalStack">
       <widget class="QTableView" name="govTableView">
       </widget>
       <widget class="QWidget" name="emptyPage">
        <layout class="QVBoxLayout" name="emptyLayout">
         <item>
          <spacer name="emptySpacerTop">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="emptyLabel">
           <property name="text">
            <string>No active proposals on the network.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="emptySpacerBottom">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_bottom">
       <item>
        <widget class="QPushButton" name="btnCreateProposal">
         <property name="text">
          <string>Create Proposal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResumeProposal">
         <property name="text">
          <string>Resume Proposal</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_bottom">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_count">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_count_2">
           <property name="text">
            <string>Proposal Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="countLabel">
           <property name="text">
            <string notr="true">0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
